usaspending_api/search/v2/views/search_elasticsearch.py

```diff
@@ -17,10 +17,14 @@
 from usaspending_api.common.validator.award_filter import AWARD_FILTER, AWARD_FILTER_NO_RECIPIENT_ID
 from usaspending_api.common.validator.pagination import PAGINATION
 from usaspending_api.common.validator.tinyshield import TinyShield
-from usaspending_api.search.v2.elasticsearch_helper import spending_by_transaction_count
+from usaspending_api.search.v2.elasticsearch_helper import (
+    get_number_of_unique_terms_for_transactions,
+    spending_by_transaction_count,
+)
 from usaspending_api.search.v2.es_sanitization import es_minimal_sanitize
 from usaspending_api.search.v2.elasticsearch_helper import spending_by_transaction_sum_and_count
 from usaspending_api.awards.v2.lookups.elasticsearch_lookups import TRANSACTIONS_SOURCE_LOOKUP, TRANSACTIONS_LOOKUP
+from elasticsearch_dsl import Q as A
```

Comment:
Any particular reason you're importing the `Q` shortcut as `A` instead of just importing the `A` shortcut instead? I understand they can operate in a very similar way, but I wasn't sure if there was a specific reason for using `Q` here.

Reply:
Following what existing code does.

Reply:
Ahh, I see where we're doing that in a few places now. As our existing code proves, for our use case there's no issue with using `Q()` instead of `A()` since `Q()` is meant to represent a generic query including aggregate queries. No issues with using it here.

**Note**: This is probably something we should clean up in a future ticket, because queries and aggregates are different and you can't _always_ use a query in place of an aggregation.